src/js/components/RangeSelector/EdgeControl.js

```diff
@@ -29,7 +29,11 @@ const EdgeControl = forwardRef(
     const { theme } = useThemeValue();
     const [focus, setFocus] = useState(false);
     const { cursor, fill } = DIRECTION_PROPS[direction];
-    const size = parseMetricToNum(theme.global.spacing) / 2;
+    const themeSize = theme.rangeSelector?.edge?.size;
+    const size =
+      typeof themeSize === 'string'
```

Comment:
Do we need this check? Can we enforce this in typescript definition and propTypes?

Reply:
yes we dont need the check